Makefile

```diff
@@ -54,3 +55,17 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+builder-rpm:
+	cd $(WD)/packages && docker build -t calico-mesos-builder .
```

Comment:
It gets confusing with the all the `cd` commands in my make instructions so I going to keep the WD variable for now.